nemo/collections/asr/parts/numba/rnnt_loss/rnnt.py

```diff
@@ -0,0 +1,220 @@
+# Copyright (c) 2021, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2018-2019, Mingkun Huang
```

Comment:
have used this code here?

Reply:
Can we mention and give link to the source on github?

Reply:
> have used this code here?

The current rnnt loss we compile in the container is actually this implementation :D 

> Can we mention and give link to the source on github?

Yes good point.

Reply:
It is always nice to have a link to the source if we have used code from somewhere other than just the license.